content/cockpit-134.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Cockpit 134
+author: Dominik Perpeet
+date: 2017-03-09 18:00
+tags: cockpit, linux
+slug: cockpit-134
+category: release
+summary: Cockpit is now fully translatable
+comments: true
+---
+
+Cockpit is the [modern Linux admin interface](http://cockpit-project.org/). We release regularly.
+Here are the release notes from version 134.
+
+### Login page is now translatable
+
+The majority of Cockpit has long been translatable. Since the login page is a bit of a special case, it required some
+special attention to make that also play well with the language display settings. Take a look at the video below for a demo.
+Check out the Zanata link below, it's very easy to contribute translations to Cockpit.
+At this time our top 3 translations (>90%) are Polish (pl), Ukranian (uk) and Chinese (zh-CN). Every bit of help
+here is greatly appreciated and a big thank you to our contributors!
+
+[Cockpit translations](https://fedora.zanata.org/project/view/cockpit?dswid=-8859)
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/_4jQ6NYSL-c?ecver=1" frameborder="0" allowfullscreen></iframe>
+
+### Show message of the day on the System page
+
+Cockpit now supports one more feature it was previously missing compared to the command line login: the ```System```
+page now shows the current message of the day. When dismissed by clicking on the ```X``` close button, the message
+is hidden until it changes.
+
+![Message of the day](http://cockpit-project.org/blog/images/cockpit-motd.png)
+
+### Expose fewer system service actions
+
+Cockpit previously exposed systemd unit actions such as ```Reload or Try Restart``` which are primarily intended for
+scripts. The action list now only show actions which are useful for interactive human usage: ```Start```, ```Stop```,
```

Comment:
Too many "actions" for my taste, so how about "Now it only shows actions which are useful for interactive human usage:"?